managed/services/telemetry/config.default.yml

```diff
@@ -531,3 +531,43 @@ telemetry:
     data:
       - metric_name: "k8s_clusters_count"
         column: "count"
+
+  # Advisor
+  - id: AdvisorChecksDownloadedCount
+    source: VM
+    query: pmm_managed_checks_checks_downloaded_total
+    transform:
+      type: JSON
+      metric: "advisor_checks_downloaded"
+    summary: "Advisor - number of downloaded checks per name"
+    data:
+      - metric_name: "check_name"
+        label: "check_name"
+      - metric_name: "count"
+        value: 1
+
+  - id: AdvisorChecksExecutedCount
+    source: VM
+    query: pmm_managed_checks_scripts_executed_total
+    transform:
+      type: JSON
+      metric: "advisor_checks_executed"
+    summary: "Advisor - number of executed checks per name"
+    data:
+      - metric_name: "check_name"
+        label: "check_name"
+      - metric_name: "count"
+        value: 1
+
+  - id: AdvisorChecksFailedCount
+    source: VM
+    query: pmm_managed_checks_alerts_generated_total
+    transform:
+      type: JSON
+      metric: "advisor_checks_failed"
```

Comment:
That sounds more like check failed itself, not that it detected some problem and fired an alert. I think it makes sense to name it more obvious.